### ISSUE TITLE
Hover-over help text in records

### ISSUE THREAD

**Issue by allysonlister** [FAIRsharing/fairsharing.github.io#2295]
A user has recently suggested the following:

> May I suggest a hover-over info be added to the headings on the page such as "Reference URL" could mean link to a page where you tell us about linked things or it could be something you want people to cite when refering to your resource. 
I think each of those heading items on the left of the page would benefit from readily available descriptions about them (either hover-over or link-out).

So, essentially they're suggesting that, just as we have "?" in the edit interface, we should have some way of providing short descriptive sentences about each of our fields in our record view. Having lots of "?" might clutter up the interface, but could we have a sentence appear if a user hovers over the text itself?

To get a first pass at the help text, you could look at the existing help text for the edit ui and adjust it appropriately for someone viewing the record. Thanks!

**Comment by knirirr** [comment]
Are you expecting there to be tooltips for everything where there is one for editing?
Personally, I'd prefer the consistent appearance of a tooltip icon for both editing and viewing, or neither.

**Comment by allysonlister** [comment]
Yes, that's right, tooltips for every field where there is one in editing. we already have some tooltips with "?" in the record viewing, e.g. cross references in the additional information section. So, that style, but more prevalent across the record.

Thanks!

**Comment by knirirr** [comment]
Do you expect to see tooltips for the general information section, e.g. name, abbreviation, description etc.
In the edit interface they are generally present but not for status or registry (which I would have thought needed them).

**Comment by allysonlister** [comment]
Yes, tooltips for the general information section, all items if possible. Once we've done that we can look at the other sections and see where it makes sense to add them

**Comment by knirirr** [comment]
This PR covers the general information section: https://github.com/FAIRsharing/fairsharing.github.io/pull/2305

Further PRs will be necessary at a later date (once Vue3 is dealt with) to complete tooltips for other record sections.
